README.md

```diff
@@ -33,13 +33,13 @@ $ deactivate
 
 More information [here](https://docs.python.org/3/library/venv.html)
 
-You will also need to initialize the `toynet_mininet` submodule and its `mininet` submodule:
+
+From there, make sure to build the toynet docker image from the `toynet_mininet` directory and tag it with the same tag you use in the below environment variable for `TOYNET_IMAGE_TAG`. First you will need to download mininet. You can do this by initializing the submodule.
 ```
+$ cd toynet_mininet
 $ git submodule update --init --recursive
 ```
-
-From there, make sure to build the toynet docker image from `toynet_mininet` and tag it with the same tag you use in the below environment variable for `TOYNET_IMAGE_TAG`.
-See the `toynet_mininet` submodule for instructions on building the image. The default way, which will work with the Makefile here, is to go into the `toynet_mininet` submodule and run:
+You will then need to build the image.  The default way, which will work with the Makefile here, is to go into the toynet_mininet submodule and run:
 ```
 $ make test-image
```

Comment:
Should we update this to:
$ make prod-image

Reply:
I think its ok in the current order that it's in. This guide assumes they're developing toynet-flask so it would make more sense to have `make test-image` first IMO.